README.md

```diff
@@ -15,34 +15,54 @@ Docker container for Ampache, a web based audio/video streaming application and
 To run official builds from docker hub you can run these commands:
 
 To run the current Ampache master (stable) branch
+
 ```bash
 docker run --name=ampache -d -v /path/to/your/music:/media:ro -p 80:80 ampache/ampache
 ```
 
 To run the current Ampache develop branch
+
 ```bash
 docker run --name=ampache -d -v /path/to/your/music:/media:ro -p 80:80 ampache/ampache:develop
 ```
 
 The develop tag is set up to use git updates so you don't have to rebuild your images to stay up to date with development.
 
+### With docker-compose
+
+Included in the repository is a simple docker-compose.yml file to get started. To use simply do:
+
+```bash
+docker-compose up -d
+```
+
+The first time you run the container, you will also need to set the correct permissions on the configuration folder:
+
+```bash
+chown www-data:www-data conf -R
```

Comment:
This can't be done reliably on the host, as either the www-data user might not exist or its UID might not match. A workaround would be to chown it to uid 33, although it isn't ideal either.

Reply:
Seems like the most reliable method would be to create a named volume instead, then just modify the file from there (`/var/lib/docker/volumes/vol-name/_data/`). I think this would work best, I'll have to test it out though.

Reply:
do you want to test that out first?

Reply:
Why not simply do UID/GID mapping with an envvar, like this https://github.com/tomsquest/docker-radicale/blob/master/docker-entrypoint.sh (and create a volume)?

Reply:
I am going to look into that, and will close out this pull request because I have tons more updates I'd like to submit.